src/common/node/fileSystem.ts

```diff
@@ -1,28 +1,33 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for details.
 
-import * as fs from "fs";
+import * as nodeFs from "fs";
 import * as path from "path";
 import * as Q from "q";
 
 export class FileSystem {
+    private fs: typeof nodeFs;
```

Comment:
Your naming pattern is inconsistent. In the previous file, you named the member `childProcess`, which is different from the actual node module (`child_process`). Here, you used exactly the name of the node module (`fs`) for the member and you renamed the import. We need better naming conventions for these situations.